### ISSUE TITLE
Cant open the controls menu

### ISSUE THREAD

**Issue by DarkRazer1** [ShadowMario/FNF-PsychEngine#16012]
### Describe your bug here.

Whenever I try to open the controls menu, it crashes and I don't know why. (I'm on version 0.7.1 BTW)

### Command Prompt/Terminal logs (if existing)

```bash
Null Object Reference

Called from options.ControlsSubState.addKeyText (options/ControlsSubState.hx line 182)
Called from options.ControlsSubState.createTexts (options/ControlsSubState.hx line 155)
Called from options.ControlsSubState.new (options/ControlsSubState.hx line 110)
Called from options.OptionsState.openSelectedSubstate (options/OptionsState.hx line 21)
Called from options.OptionsState.update (options/OptionsState.hx line 101)
Called from flixel.FlxState.tryUpdate (flixel/FlxState.hx line 239)
Called from flixel.FlxGame.update (flixel/FlxGame.hx line 749)
Called from flixel.FlxGame.step (flixel/FlxGame.hx line 679)
Called from flixel.FlxGame.onEnterFrame (flixel/FlxGame.hx line 558)
Called from openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 426)
Called from openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1398)


 === 

Commit: 01dc8ab0ee2a7c630e5e4525d7c3b7b92768df1c
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

Nope

**Comment by Shadow06735** [comment]
Double check if that state is there

**Comment by DarkRazer1** [comment]
Ok, I installed 1.0, and... the glitch didn't happen. It only happens on 0.7.1 for some reason

**Comment by DarkRazer1** [comment]
![Capture](https://github.com/user-attachments/assets/1611dd1e-822c-4293-9dfc-1f934ef6db78)
(Screenshot from 0.7.1)

**Comment by Shadow06735** [comment]
> ![Capture](https://private-user-images.githubusercontent.com/138380836/393468182-1611dd1e-822c-4293-9dfc-1f934ef6db78.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM1NDQ5NTYsIm5iZiI6MTczMzU0NDY1NiwicGF0aCI6Ii8xMzgzODA4MzYvMzkzNDY4MTgyLTE2MTFkZDFlLTgyMmMtNDI5My05ZGZjLTFmOTM0ZWY2ZGI3OC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwN1QwNDEwNTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jY2RiNzA3Zjg1NDNlMzgyZGFhMDIxODZjNjEzOTdhMDFiOTU3MWE2OTY0NTY3Mzg3MTk3OWZiYTZkNGIzZTEzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wI2qAuKDWLBmSxlkY2z59hl7cs18Is_a-5AL67TokU8) (Screenshot from 0.7.1)

yeah, 0.7.1 is really buggy so i wouldn't recommend using that. good thing you found a solution tho, i think this can be closed now
